.cbmc-batch/jobs/aws_cryptosdk_sign_header/README.md

```diff
@@ -0,0 +1,18 @@
+# Memory safety proof for aws_cryptosdk_sign_header
```

Comment:
Standard question: is this intrinsic to the harness, or the function?

Reply:
The function -- I've added additional documentation to both READMEs.